demos/tutorial/graduation_case_2.cpp

```diff
@@ -50,8 +50,8 @@ int main(int argc, char* argv[])
     fs::path path        = fs::current_path();
     std::string filename = "graduation_case_2";
 
-    app.add_option("--min-level", min_level, "Minimum level of the mesh generator")->capture_default_str();
-    app.add_option("--max-level", max_level, "Maximum level of the mesh generator")->capture_default_str();
+    app.add_option("--minimum-level", min_level, "Minimum level of the mesh generator")->capture_default_str();
+    app.add_option("--maximum-level", max_level, "Maximum level of the mesh generator")->capture_default_str();
```

Comment:
same here. Knowing that `min-level` and `max-level` are options already provided by samurai.

Reply:
Because to access to `--min-level` value we need `samurai::args::min_level` and we want to provide the same access to all arguments in this tutorial.